tests/data/dataset_readers/gqa_test.py

```diff
@@ -0,0 +1,107 @@
+from typing import Dict
+
+import torch
+from torch import Tensor
+
+from allennlp.common.testing import AllenNlpTestCase
+from allennlp.data import Batch, Vocabulary
+from allennlp.data.dataset_readers import GQAReader
+from allennlp.data.image_loader import DetectronImageLoader
+from allennlp.data.tokenizers import WhitespaceTokenizer
+from allennlp.data.token_indexers import SingleIdTokenIndexer
+from allennlp.modules.vision.grid_embedder import NullGridEmbedder
+from allennlp.modules.vision.region_detector import RegionDetector
+
+
+class FakeRegionDetector(RegionDetector):
```

Comment:
I think you can use the `RandomRegionDetector` instead of having to specify a fake one here. It's basically the same code logic.